docker.config.rockpi

```diff
@@ -7,7 +7,8 @@
     ]},
     {blockchain,
     [
-      {key, {ecc, [{key_slot, 0}, {onboarding_key_slot, 0}, {bus, "i2c-7"}]}}
+      {key, {ecc, [{key_slot, 0}, {onboarding_key_slot, 0}, {bus, "i2c-7"}]}},
+      {blocks_to_protect_from_gc, 5000}
```

Comment:
These are set to 5000 currently so maybe just change to 4000 as per @KevinWassermann94 and then looks good to go to me for testing

Reply:
Perfect, changes have been applied. I set it to 4,000 in all 3 configs